Integrations/ThreatConnect/ThreatConnect.yml

```diff
@@ -1996,6 +2026,191 @@ script:
+      description: The date that the group was added.
+      type: Date
+    - contextPath: TC.Group.EventDate
+      description: Event date.
+      type: Date
+    - contextPath: TC.Group.ID
+      description: Group ID.
+      type: Number
+    - contextPath: TC.Group.Name
+      description: Group name.
+      type: String
+    - contextPath: TC.Group.Owner.id
+      description: The owner ID.
+      type: Number
+    - contextPath: TC.Group.Owner.name
+      description: The owner name
+      type: String
+    - contextPath: TC.Group.Owner.type
+      description: The type of the owner
+      type: String
+    - contextPath: TC.Group.Status
+      description: Group status.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: The group type to return. Can be "adversaries", "campaigns", "documents",
+        "emails", "events", "incidents", "intrusionSets", "reports", "signatures",
+        or "threats".
+      isArray: false
+      name: group_type
+      predefined:
+      - adversaries
+      - campaigns
+      - documents
+      - emails
+      - events
+      - incidents
+      - intrusionSets
+      - reports
+      - signatures
+      - threats
+      required: true
+      secret: false
+    - default: false
+      description: Group ID.
+      isArray: false
+      name: group_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve a Group’s Attributes
+    execution: false
+    name: tc-get-group-attributes
+    outputs:
+    - contextPath: TC.Group.DateAdded
+      description: The date that the group was added.
+      type: Date
+    - contextPath: TC.Group.Displayed
+      description: Is the attribute displayed.
+      type: Boolean
+    - contextPath: TC.Group.ID
+      description: Attribute ID.
+      type: Number
+    - contextPath: TC.Group.LastModified
+      description: Attribute last modified date.
+      type: Date
+    - contextPath: TC.Group.Type
+      description: Attribute type.
+      type: String
+    - contextPath: TC.Group.Value
+      description: Attribute value
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      description: The group type to return. Can be "adversaries", "campaigns", "documents",
+        "emails", "events", "incidents", "intrusionSets", "reports", "signatures",
+        or "threats".
+      isArray: false
+      name: group_type
+      predefined:
+      - adversaries
+      - campaigns
+      - documents
+      - emails
+      - events
+      - incidents
+      - intrusionSets
+      - reports
+      - signatures
+      - threats
+      required: true
+      secret: false
+    - default: false
+      description: Group ID.
+      isArray: false
+      name: group_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve a Group’s Security Labels
+    execution: false
+    name: tc-get-group-security-labels
```

Comment:
same here with context, need to add security labels list, under it all security labels